### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by Neenu1995**
b/218359798

### CODE REVIEW

**Review by chanseokoh — APPROVED** [review]

File: .readme-partials.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 custom_content: |
   ### Compute alpha to beta migration
   
-  From version 1.6.0-beta `google-cloud-compute` will have the following feature changes: 
+  Java compute library is GA from version 1.7.0 and is completely distinct from the previous 1.x.0 and 0.x.0 versions. From version 1.6.0-beta `google-cloud-compute` will have the following feature changes:
```
Comment (vam-google) [inline .readme-partials.yaml:4]: "complete distinct" is too strong wording here, also we probably need separate  0.x.x and 1.x.x here.

Essentially it is:
- backwards incompatible with 0.x.x
- is incompatible with 1.5.x-beta and prior in a following way: (and here you keep the description which is already there about LRO and Operation differences).

Comment (Neenu1995) [reply .readme-partials.yaml]: done

File: .readme-partials.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 custom_content: |
   ### Compute alpha to beta migration
   
-  From version 1.6.0-beta `google-cloud-compute` will have the following feature changes: 
+  Java compute library is GA from version 1.7.0 and backwards incompatible with 0.x.x. Also it is incompatible with 1.5.x-alpha and prior in a following way: 
   - Everything except polling methods which used to return `Operation` now returns `OperationFuture`. 
   - Library automatically polls Operation status.
   - `Operation op = client.myMethod(args)` should be replaced with `OperationFuture<Operation, Operation> opFuture = client.myMethodAsync(args);` 
-  - Manual polling now just calls `opFuture.get()` and wait for it to complete polling. 
+  - Polling is now done automatically, manual polling is no longer required. Calling opFuture.get() will wait for automatic polling to complete. It will return the result of the long running operation once the operation is completed on the server side or throw an exception if an error occurs during polling.
```
Comment (vam-google) [inline .readme-partials.yaml:8]: please escape `opFuture.get()`

**Review by vam-google — APPROVED** [review]
LGTM with one nit

File: .readme-partials.yaml (hunk 2 above)
Comment (Neenu1995) [reply .readme-partials.yaml]: done

**Pull request merged**